pkgs/development/ocaml-modules/ocaml-r/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildDunePackage, pkg-config, configurator, stdio, R }:
+
+buildDunePackage rec {
+  pname = "ocaml-r";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "pveber";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "09gljccwjsw9693m1hm9hcyvgp3p2fvg3cfn18yyidpc2f81a4fy";
+  };
+
+  preConfigure = ''
+    substituteInPlace stubgen/stubgen.ml --replace 'failwithf "not supported: %s" name ()' 'sprintf "(* not supported: %s *)" name'
+  '';
+
+  buildInputs = [ configurator pkg-config R stdio ];
+
+  meta = {
+    description = "OCaml bindings for the R interpreter";
+    inherit (src.meta) homepage;
+    license = lib.licenses.gpl3;
+  };
```

Comment:
Feel free to add me as maintainer if you want.

Reply:
Thanks a lot!